tests/factories.py

```diff
@@ -125,6 +152,31 @@ class SoftDeleteTestModelFactory(
     timedelta = factory.Faker("time_delta")
     json_field = factory.Faker("pydict", allowed_types=[str, int, float])
     date_range = saritasa_sqlalchemy_tools.DateRangeFactory()
+    file = saritasa_s3_tools.factory.S3ImageFileField(
+        s3_config="files",
+        s3_region="us-west-1",
+        bucket="fastapi-rest-framework-files",
+        access_key_getter=lambda: botocore.credentials.Credentials(
+            access_key="root",
+            secret_key="rootroot",
+        ),
+        s3_endpoint_url_getter=lambda: "https://localhost.localstack.cloud:4566",
+    )
+    files = factory.List(
+        [
+            saritasa_s3_tools.factory.S3ImageFileField(
+                s3_config="files",
+                s3_region="us-west-1",
+                bucket="fastapi-rest-framework-files",
+                access_key_getter=lambda: botocore.credentials.Credentials(
+                    access_key="root",
+                    secret_key="rootroot",
+                ),
+                s3_endpoint_url_getter=lambda: "https://localhost.localstack.cloud:4566",
```

Comment:
Maybe it's better to create `constants.py` in root to have access to this url? It is used in example app and in tests a few times